.github/workflows/build-with-kokkos.yml

```diff
@@ -47,7 +49,7 @@ jobs:
       - name: Build Kokkos Tools, enabling examples
         run: |
           cmake -S . --preset=${{ matrix.preset }}
-          cmake --build --preset=${{ matrix.preset }}
+          cmake --build --preset=${{ matrix.preset }} --verbose
```

Comment:
Did you mean to commit that change?

Reply:
Indeed, no. I reverted that change :heavy_check_mark: